host/src/lib/HelperFunctions.jsx

```diff
@@ -283,31 +284,31 @@ export const determineAnswerType = (answer) => {
     return AnswerType.STRING;
   }
 };
-
 /**
```

Comment:
annoying formatting here (probably on my end) but here's what's happening:

`checkEqualityWithPrevAnswer` is being removed. That equality check now happens in the main function `checkEqualityWithOtherAnswers` instead of offloading it to another function. 

`checkExpressionEquality` is a new function that is added that exclusively handles the checking of expressions (so that we can use `mathjs` `symbolicEqual`

so `checkEqualityWithPrevAnswer` and `checkExpressionEquality` are two separate functions with unique purposes, one isn't being replace with another